doc/compose.md

```diff
@@ -33,10 +33,16 @@ This is a beta feature, please [join our community](https://ploomber.io/communit
 let us know how we can improve it!
 ```
 
-JupySQL allows you to break queries into multiple cells, simplifying the process of
-building large queries.
+Tired of wrangling complex, large-scale SQL queries? JupySQL is here to revolutionize your experience with an intuitive, user-friendly package specifically designed to help you manage and organize large SQL queries in Jupyter Notebooks.
```

Comment:
As a general rule, we keep the tone in our docs non sales-y, and more direct.

Reply:
Noted. Resolve in https://github.com/ploomber/jupysql/pull/300/commits/50219a0c10c5aa8bc64bbbd849e4e41f945ade1d